Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/censors/submission/OtpCensor.kt

```diff
@@ -0,0 +1,29 @@
+package de.rki.coronawarnapp.bugreporting.censors.submission
+
+import dagger.Reusable
+import de.rki.coronawarnapp.bugreporting.censors.BugCensor
+import de.rki.coronawarnapp.srs.core.storage.SrsSubmissionSettings
+import kotlinx.coroutines.flow.last
+import javax.inject.Inject
+
+@Reusable
+class OtpCensor @Inject constructor(
+    private val srsSubmissionSettings: SrsSubmissionSettings,
+) : BugCensor {
+
+    override suspend fun checkLog(message: String): BugCensor.CensorContainer? {
+        var container = BugCensor.CensorContainer(message)
+
+        val otp = srsSubmissionSettings.otp.last()
```

Comment:
this would censor only OTP once it is saved in the repo, but before than I won't be censored

Reply:
censor otp on fetch and renewal